tools/tracegen/Port.cpp

```diff
@@ -30,8 +30,10 @@
 #define J9FILE_UNC_EXTENDED_LENGTH_PREFIX (L"\\\\?\\")
 #else /* defined(WIN32) */
 #include <sys/types.h>
-#if !defined(OMRZTPF)
+#if defined(AIXPPC)
 #include <sys/statvfs.h>
+#elif (defined(LINUX) && !defined(J9ZTPF)) || defined(OSX)
```

Comment:
OSX needs sys/statfs.h. Compile failed for Travis build on my fork, but passed on eclipse/omr validation build for same push. Scratching head, but code inspection clearly shows problems with statvfs and statfs usage vis-a-vis includes and build flags. I believe my changes are correct.

Only explanation I can think of is that OSX builds for eclipse/omr validation include sys/statfs.h indirectly, but for briggrs/omr they do not.